### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by efrapp**
Creating pull request for coding review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
```
Comment (heyapricot) [inline index.html]: Why is this empty?

File: style.css [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+  background-image: url("img/ecosystem_logos.png");
+  background-position: 0 0;
+  background-repeat: no-repeat;
+  background-size: 458px 83px;
+  display: block;
+  height: 34px;
+  width: 118px;
+}
+
+#title ul{
```
Comment (heyapricot) [inline style.css]: You used this same selector on line 58. Why not grouping them?

File: style.css [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+ height: 34px;
+ width: 320px;
+ min-width: 160px;
+ padding-left: 8px;
+ padding-right: 6px;
+ border: 1px solid #C1C5C8;
+ margin: 0 auto;
+ text-align: center;
+ display: block;
+ width: 100%;
```
Comment (heyapricot) [inline style.css]: This property overwrites the change from line 136

File: style.css [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+ background-color: #ECEEF1;
+ font-size: 11px;
+ color: #8d9096;
+}
+.footer{
+ margin: 0;
+ padding: 0;
+ line-height: 1.5;
+ outline: none;
+ -ms-box-sizing: border-box;
```
Comment (heyapricot) [inline style.css]: I don't think this is a valid property

File: style.css [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+ font-size: 14px;
+ line-height: 25px;
+}
+#crt_btn{
+ margin-top: 10%;
+ width: 100%;
+ height: 44px;
+ padding: 8px;
+ padding-left: 21px;
+ padding-right: 21px;
```
Comment (heyapricot) [inline style.css]: Can replace the padding lines with `padding: 8px 21px`

File: style.css [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+ background: #eceef1;
+}
+
+body{
+  overflow-x: hidden;
+}
+
+/* Top bar */
+header{
+  /* background: red; */
```
Comment (heyapricot) [inline style.css]: Please avoid commenting code on final submissions

**Review by heyapricot — COMMENTED** [review]
Nice work! Looks really good.
Please address the comments and after that you are ready to move on. No need from my side for further review. 
Tips: 
add .DS_Store to the .gitignore file. This is a file made by default by OSX. No need to keep track of it.
For every HTML/CSS projects please check your files with the [W3 Validator](https://validator.w3.org/) and the [CSS Linter](http://csslint.net/)

File: index.html (hunk 1 above)
Comment (efrapp) [reply index.html]: Done

File: style.css (hunk 2 above)
Comment (efrapp) [reply style.css]: Merged in one selector.

File: style.css (hunk 3 above)
Comment (efrapp) [reply style.css]: I removed this one because this breaks the style if I remove the other one.

File: style.css (hunk 4 above)
Comment (efrapp) [reply style.css]: Removed.

File: style.css (hunk 5 above)
Comment (efrapp) [reply style.css]: Changed.

**Pull request merged**